src/app/shared/plnkr/plunker-data.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Plunker data
+ *   with information about Component name, selector, files used in example, and path to examples
+ */
+export class PlunkerData {
```

Comment:
I think this is fine for this CL, but this stuff isn't actually related to plunker at all, it should be called ExampleData and should probably live in the material2 repo eventually.